<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vPackageB</class>
 <widget class="QWidget" name="vPackageB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="sortedGroup">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>251</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="sortedTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Simulated Dream Vacation</string>
    </property>
   </widget>
   <widget class="QListWidget" name="sortedTeamList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>230</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>125</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:orange;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;
color: black;</string>
    </property>
    <property name="text">
     <string>Start Vacation!</string>
    </property>
   </widget>
   <widget class="QLabel" name="distanceLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>444</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total Distance:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="distanceLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>-6</y>
     <width>559</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Detroit Pistons Package</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ccccff;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;
color: black;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mudkipLabel">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>42</y>
     <width>265</width>
     <height>511</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/verticalMudkip.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
